src/types.d.ts

```diff
@@ -1,4 +1,133 @@
-declare module "notifications-node-client";
+declare module "notifications-node-client" {
```

Comment:
type declaration for notify client. Taken from: https://github.com/alphagov/notifications-node-client/pull/154